fog/view/server/tests/unary_smoke_tests.rs

```diff
@@ -206,9 +206,9 @@ fn test_view_integration(view_omap_capacity: u64, store_count: usize, blocks_per
         .sort_by_key(|rng_record| rng_record.ingest_invocation_id);
     assert_eq!(result.rng_records[0].pubkey, pubkey1);
     assert_eq!(result.rng_records[1].pubkey, pubkey2);
-    assert_eq!(result.tx_out_search_results.len(), 1);
+    assert_eq!(result.fixed_tx_out_search_results.len(), 1);
```

Comment:
:wrench: Similar to streaming_smoke_tests this should probably still be checking `tx_out_search_results`

Reply:
Done.